packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -269,7 +270,7 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
     // Now validate the relations
     await this._validateRelationsContent(ceramic, model, content)
     if (!genesis && payload) {
```

Comment:
I think the validation could be completely bypassed when there is no existing content, rather than needing additional logic there
```suggestion
    if (!genesis && payload && oldContent != null) {
```